src/main/java/edu/kis/vh/nursery/containers/IntArrayStack.java

```diff
@@ -1,35 +1,41 @@
-package edu.kis.vh.nursery;
+package edu.kis.vh.nursery.containers;
 
-public class IntArrayStack {
+import edu.kis.vh.nursery.IStackInterface;
 
-	private static final int EMPTY = -1;
+public class IntArrayStack implements IStackInterface{
+
+	private static final int EMPTY = 0;
 	private static final int SIZE = 12;
-	private static final int FULL = SIZE - 1;
+	private static final int FULL = SIZE -1;
 
 	private int[] numbers = new int[SIZE];
 
 	private int total = EMPTY;
 
+	@Override
 	public void countIn(final int in) {
 		if (!isFull())
-			numbers[++total] = in;
+			numbers[total++] = in;
```

Comment:
Changing EMPTY value requires such fixes? It does not make sense. Incorrect fix.